package.json

```diff
@@ -42,7 +42,7 @@
     "@angular/language-service": "4.4.3",
     "@types/jasmine": "~2.5.53",
     "@types/jasminewd2": "~2.0.2",
-    "@types/node": "~6.0.60",
+    "@types/node": "^6.0.60",
```

Comment:
Same as the others. keep it if needed, but if not then let's leave it.

Reply:
This is in dev dependencies and it will have a fix coming for unit testing that will be picked up soon.

Reply:
Until this is picked up it will we will need to change the type 'NodeRequire' to 'any' at node_modules/@types/node/index.d.ts (87,13)